test/e2e/common/setup-kubestellar.sh

```diff
@@ -109,14 +104,25 @@ esac
 : Install the core-chart
 :
 
-pushd "${SRC_DIR}/../../.."
-make kind-load-image
-make install-local-core-chart \
-  INSTALL_KUBEFLEX=false \
-  ITS_NAME=its1 \
-  DEFAULT_WDS_NAME=wds1 \
-  KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY=$KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY
-popd
+if [ "$use_release" = true ] ; then
+  kubestellar_latest_release=$(curl -sL https://github.com/kubestellar/kubestellar/releases/latest | grep "</h1>" | tail -n 1 | sed -e 's/<[^>]*>//g' | xargs)
+  helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+    --version ${kubestellar_latest_release//v} \
+    --set kubeflex-operator.install=false \
+    --set-json='ITSes=[{"name":"its1"}]' \
+    --set-json='WDSes=[{"name":"wds1"}]' \
+    --set verbosity.kubestellar=${KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY} \
+    --set verbosity.transport=${TRANSPORT_CONTROLLER_VERBOSITY}
+else
+  pushd "${SRC_DIR}/../../.."
+  make kind-load-image
+  make install-local-core-chart \
+    INSTALL_KUBEFLEX=false \
+    ITS_NAME=its1 \
+    DEFAULT_WDS_NAME=wds1 \
+    KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY=$KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY
```

Comment:
The `install-local-core-chart` target uses `KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY` to set _both_ the kubestellar controller-manager verbosity _and_ the OCM transport controller verbosity. This does not line up with the separate treatment in this script. IMHO the best resolution would be to update the `install-local-core-chart` target to be more articulate about verbosity controls.

Reply:
If you see my comment above, I was thinking that it is probably better to completely remove the target and just directly use helm with the correctly named args

Reply:
I implemented the change in #2473.
We will need to rebase.

Reply:
I removed the target in #2473